src/ImageBuilder/custom/New-AzImageBuilderTemplateDistributorVersioningObject/New-AzImageBuilderTemplateDistributeVersionerLatestObject.ps1

```diff
@@ -0,0 +1,48 @@
+
+# ----------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Code generated by Microsoft (R) AutoRest Code Generator.Changes may cause incorrect behavior and will be lost if the code
+# is regenerated.
+# ----------------------------------------------------------------------------------
+
+<#
+.Synopsis
+Create an in-memory object for DistributeVersionerLatest.
+.Description
+Create an in-memory object for DistributeVersionerLatest.
+
+.Outputs
+Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Models.Api20220701.DistributeVersionerLatest
+.Link
+https://learn.microsoft.com/powershell/module/Az.ImageBuilder/new-azimagebuildertemplatedistributeversionerlatestobject
+#>
+function New-AzImageBuilderTemplateDistributeVersionerLatestObject {
+    [OutputType('Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Models.Api20220701.DistributeVersionerLatest')]
+    [CmdletBinding(PositionalBinding=$false)]
+    Param(
+
+        [Parameter(HelpMessage="Major version for the generated version number. Determine what is `"latest`" based on versions with this value as the major version. -1 is equivalent to leaving it unset.")]
+        [int]
+        $Major
+    )
+
+    process {
+        $Object = [Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Models.Api20220701.DistributeVersionerLatest]::New()
```

Comment:
can this be done by a model-cmdlet? Or why to support a new object cmdlet with only one parameter?

Reply:
Commands generated through model-cmdlet, and for ease of use, I set the unique value of the parameter Scheme to the default value.